src/components/Store/Store.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import Nav from '../Nav/Nav';
+import StoreNav from '../StoreNav/StoreNav';
+import Footer from '../Footer/Footer';
+import FilterNav from '../FilterNav/FilterNav';
+import GameCard from '../GameCard/GameCard';
+import games from '../../data-structures/games';
+import FilterError from '../FilterError/FilterError';
+import { rightNavButtons, leftNavButtons } from './store.config';
+import styles from './Store.module.css';
+
+const Store = () => {
+  const [displayedGames, setDisplayedGames] = React.useState(games);
+  const [filter, setFilter] = React.useState('none');
+  // first element of the error var will always be the main error message
+  // subsequent elements will represent notes or additional messages.
+  const [error, setError] = React.useState(null);
+
+  const handleWishList = (gameName, action) => {
+    for (let game of games) {
+      if (game.name === gameName) {
+        action === 'add' ? (game.wishList = true) : (game.wishList = false);
```

Comment:
static data should not be modified directly, if you need to modify data it should be added as state